schemas/test/en/test_unit_label_hover.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Why do you need a schema for this?. Use the existing units one?

Reply:
Just cause I wanted it separate, removed now.